CHANGELOG.md

```diff
@@ -1,3 +1,47 @@
+- Changed: ``layouts`` should not be defined in build and client scripts
+  anymore. This method will be deprecated in a future version.
+  Instead please directly pass ``layouts`` in the ``routes`` definitions with a
+  HoC.
+  Here is an example:
+
+  ```js
+  import React, { Component } from "react"
+  import { Route } from "react-router"
+
+  import LayoutContainer from "../LayoutContainer"
+  import StatinamicPageContainer from "statinamic/lib/PageContainer"
+
+  import Page from "../layouts/Page"
+  import PageError from "../layouts/PageError"
+  import PageLoading from "../layouts/PageLoading"
+  import Homepage from "../layouts/Homepage"
+
+  class PageContainer extends Component {
+    render() {
+      const { props } = this
```

Comment:
I like to prepare myself for functional component (stateless or [stateful](https://github.com/reactjs/react-future/blob/ba5a275d08451756137559714d79f15055b9a14e/07%20-%20Returning%20State/01%20-%20Stateful%20Functions.js)) :D